src/bin/common/parse_command.ml

```diff
@@ -603,11 +581,13 @@ let mk_opts file () () debug_flags ddebug_flags dddebug_flags rule () halt_opt
     `Ok true
   end
 
-let mk_fmt_opt std_fmt err_fmt mdl_fmt
-  =
-  set_std_fmt (value_of_fmt std_fmt);
-  set_err_fmt (value_of_fmt err_fmt);
-  set_fmt_mdl (value_of_fmt mdl_fmt);
+let mk_output_channel_opt std_output deprecated_std_output err_output
+    deprecated_err_output mdl_output =
+  Options.Output.(create_channel std_output |> set_std);
+  Options.Output.(create_channel deprecated_std_output |> set_std);
+  Options.Output.(create_channel err_output |> set_err);
+  Options.Output.(create_channel deprecated_err_output |> set_err);
+  Options.Output.(create_channel mdl_output |> set_mdl);
```

Comment:
I don't think this works? Because all options return formatter names (and not options), if say `--std-output` is provided, we will set the standard channel to the value given to `--std-output` and then immediately replace that with the value from `--std-formatter` (or its default value of stdout), rendering `--std-output` useless.